src/ngraph_mark_for_clustering.cc

```diff
@@ -272,8 +272,12 @@ Status MarkForClustering(Graph* graph) {
       confirmation_functions["FloorDiv"] = SimpleConfirmationFunction();
       confirmation_functions["FloorMod"] = SimpleConfirmationFunction();
       confirmation_functions["FusedBatchNorm"] = SimpleConfirmationFunction();
-      confirmation_functions["FusedBatchNormGrad"] =
-          SimpleConfirmationFunction();
+      confirmation_functions["FusedBatchNormGrad"] = [](Node* n, bool* result) {
+        //bool tf_is_training;
```

Comment:
It's good to not have commented out code (aka `dead code`). Please remove this line.

Reply:
Removed.